pom.xml

```diff
@@ -101,6 +101,7 @@
     <maven-plugin-annotations.version>3.13.0</maven-plugin-annotations.version>
     <vertx.version>4.5.8</vertx.version>
     <zjsonpatch.version>0.3.0</zjsonpatch.version>
+    <jandex.version>3.1.8</jandex.version>
```

Comment:
I think this should be bumped to 3.2.0 to align with the current dependency in the project (sorry, this is due to the late review of the PR :pray:)